src/elements/picker/extras/entity-picker-results/EntityPickerResults.js

```diff
@@ -85,6 +97,14 @@ export class EntityPickerResults extends PickerResults {
         return '';
     }
 
+    renderTimestamp(date) {
```

Comment:
Should be injecting the `NovoLabelService`, then just `labels.formatDateWithFormat(value, format)`